pixoo/__init__.py

```diff
@@ -108,10 +109,11 @@ def clear_rgb(self, r, g, b):
     def draw_character(self, character, xy=(0, 0), rgb=Palette.WHITE):
         matrix = retrieve_glyph(character)
         if matrix is not None:
+            teiler = matrix[-1]
```

Comment:
"teiler"?
I guess English is the preferred language for this project.

Reply:
changed to divider, my mistake